src/features/comments.ts

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable */
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import * as commentsApi from '../api/comments';
+import { Comment, CommentData } from '../types/Comment';
+import { deleteComment, createComment } from '../api/comments';
```

Comment:
The `deleteComment` and `createComment` functions are imported but not used directly in the code. Consider removing these imports if they are not needed, or ensure they are used appropriately.